railties/lib/rails/test_unit/reporter.rb

```diff
@@ -15,8 +15,12 @@ def aggregated_results # :nodoc:
       filtered_results.reject!(&:skipped?) unless options[:verbose]
       filtered_results.map do |result|
         location, line = result.method(result.name).source_location
-        "bin/rails test #{location}:#{line}"
+        "bin/rails test #{relative_path_for(location)}:#{line}"
       end.join "\n"
     end
+
+    def relative_path_for(file)
+      file.sub(/^#{Dir.pwd}\/?/, '')
```

Comment:
Hm, that broke the build: https://travis-ci.org/rails/rails/jobs/63081213#L1323-L1359
I just reverted it.